client/components/editor/editor-modal-drawio.vue

```diff
@@ -40,6 +39,12 @@ export default {
     close () {
       this.activeModal = ''
     },
+    drawioBaseUrl() {
+      var params = '?embed=1&proto=json&spin=1&saveAndExit=1&noSaveBtn=1&noExitBtn=0';
+      var url = siteConfig.drawioBaseUrl
+      url += url.endsWith('/') ? '' : '/'
+      return  url + params;
+    },
```

Comment:
- Should be a computed property, not a method.
- Don't use `var`, use either const or let
- No semicolons

Could be simplified to:
```js
return `${siteConfig.drawioBaseUrl}?embed=1&proto=json&spin=1&saveAndExit=1&noSaveBtn=1&noExitBtn=0`
```